library/azure_rm_trafficmanager.py

```diff
@@ -0,0 +1,588 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2018 Hai Cao, <t-haicao@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_trafficmanager
+version_added: "2.7"
+short_description: Manage a Traffic Manager profile.
+description:
+    - Create, update and delete a Traffic Manager profile.
+
+options:
+    resource_group:
+        description:
+            - Name of a resource group where the Traffic Manager profile exists or will be created.
+        required: true
+    name:
+        description:
+            - Name of the Traffic Manager profile.
+        required: true
+    state:
+        description:
+            - Assert the state of the Traffic Manager profile. Use C(present) to create or update a Traffic Manager profile and C(absent) to delete it.
+        default: present
+        choices:
+            - absent
+            - present
+    location:
+        description:
+            - Valid azure location. Defaults to 'global'.
+    profile_status:
+        description:
+            - The status of the Traffic Manager profile.
+        default: Enabled
+        choices:
+            - Enabled
+            - Disabled
+    traffic_routing_method:
+        description:
+            - The traffic routing method of the Traffic Manager profile.
+        default: Performance
+        choices:
+            - Performance
+            - Priority
+            - Weighted
+            - Geographic
+    dns_config:
+        description:
+            - The DNS settings of the Traffic Manager profile.
+        suboptions:
+            relative_name:
+                description:
+                    - The relative DNS name provided by this Traffic Manager profile.
+            ttl:
+                description:
+                    - The DNS Time-To-Live (TTL), in seconds.
+        default:
+            relative_name: Name of the Traffic Manager profile
```

Comment:
default will be parsed to an assignment code, move this description to description